codec/src/test/java/io/netty/handler/codec/ByteToMessageDecoderTest.java

```diff
@@ -280,7 +280,7 @@ protected void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) t
             protected void decodeLast(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) throws Exception {
                 assertFalse(decodeLast);
                 decodeLast = true;
-                super.decodeLast(ctx, in, out);
+                decode(ctx, in, out);
```

Comment:
revert ?

Reply:
This is actually an intentional change since the test assumes that `super.decodeLast()` will always call `decode`, and that's no longer the case in this particular situation (because there is no additional data since the last call to decode).